tests/integration/clientv3/naming/resolver_test.go

```diff
@@ -111,3 +111,55 @@ func TestEtcdGrpcResolver(t *testing.T) {
 		break
 	}
 }
+
+func TestEtcdEndpointManager(t *testing.T) {
```

Comment:
Please use `github.com/stretchr/testify` to simplify the test case.

Reply:
Hello Benjamin,

thank you for reviewing, added asserts for the test.